test/PushNotificationsTests/APITests.cpp

```diff
@@ -91,13 +91,65 @@ namespace Test::PushNotifications
             return true;
         }
 
-        TEST_METHOD(LaunchPackagedApp)
+        TEST_METHOD(ChannelRequestUsingNullRemoteId)
         {
             wil::unique_event event = CreateTestEvent(c_testProtocolScheme_Packaged);
 
             // This is associated protocol for the MSIX installed app for launch.
             // Use the ://path to define the component you want to test.
-            Uri launchUri{ c_testProtocolScheme_Packaged + L"://this_is_a_test" };
+            Uri launchUri{ c_testProtocolScheme_Packaged + L"://ChannelRequestUsingNullRemoteId" };
+            auto launchResult = Launcher::LaunchUriAsync(launchUri).get();
+            VERIFY_IS_TRUE(launchResult);
+
+            WaitForEvent(event, m_failed);
```

Comment:
I'm seeing that we are launching an app where API testing will happen, or at least this is my expectation. Please correct me if this is wrong.

If I'm right, then where is the testing code?

Reply:
test/TestApps/PushNotificationsTestApp/main.cpp -  can you see take a look this file? Thanks.

Reply:
Ah GitHub didn't load it while reviewing. Thanks!